tests/api.v2/helpers/access/__snapshots__/createUserInvite.test.js.snap

```diff
@@ -20,7 +20,7 @@ Object {
               Hello, <br/>
               Your collaborator inviter@example.com has invited you to explore their single cell RNA-seq project in Cellenics. <br/><br/>
               Access it with the link below: <br/>
-              <a href=\\"scp.biomage.net/experiments/experimentId/data-exploration\\">scp.biomage.net/experiments/experimentId/data-exploration</a> <br/> <br/>
+              <a href=\\"mockDomainName.com/experiments/experimentId/data-exploration\\">mockDomainName.com/experiments/experimentId/data-exploration</a> <br/> <br/>
```

Comment:
in here, you should also add `config.sandboxId`, or for staged environments we use a wrong link:

<img width="789" alt="Screenshot 2022-06-17 at 10 34 39" src="https://user-images.githubusercontent.com/65414651/174249243-dc66a81a-5b02-4ff9-898f-1bf415103f5d.png">